### PR TITLE
UI Fixes and release new pricing

### PR DESCRIPTION
**Pull request by Aldrian**

### CODE REVIEW

File: app/scripts/components/academy/academy-course.components.jsx [hunk 1]
```diff
@@ -112,12 +113,12 @@ class AcademyCourse extends React.PureComponent {
 		const parts = course.content.split(/[^\#]#{2} +/g);
 		let sidebar = {};
 
-		this.courseSidebarDom.classList.remove('fixed');
+		document.querySelector('.academy-course-main-sidebar').classList.remove('fixed');
```
Comment (YoruNoHikage) [inline app/scripts/components/academy/academy-course.components.jsx]: Do we really have to do this? Why not just wrapping the previous `<div>` with `<ScrollArea />` ?

File: app/scripts/components/collection/variant.components.jsx [hunk 2]
```diff
@@ -100,16 +100,20 @@ class Variant extends React.Component {
...
-							splitButton
-							splitted={deleteSplit}
-							onClick={this.prepareDeleteOrDelete}
-							altLabel="Cancel"
-							altClick={this.cancelDelete}
-						>
-							{deleteSplit ? 'Delete' : 'Delete variant'}
-						</ContextualMenuItem>
+						{onlyVariant
+						? false
```
Comment (YoruNoHikage) [inline app/scripts/components/collection/variant.components.jsx]: `!onlyVariant && ...` is shorter

File: app/scripts/components/topbar/topbar.components.jsx [hunk 3]
```diff
@@ -338,7 +338,7 @@ class Topbar extends React.Component {
 			/>
 		);
 
-		const academyIcon = (loadingAcademyProgress || !academyProgress.lastCourse) && (
+		const academyIcon =  !academyProgress.lastCourse && (
```
Comment (YoruNoHikage) [inline app/scripts/components/topbar/topbar.components.jsx]: Extra space? 🤨

File: app/scripts/components/academy/academy-course.components.jsx (hunk 1 above)
Comment (Aldrian) [reply app/scripts/components/academy/academy-course.components.jsx]: ¯\_(ツ)_/¯

Comment (YoruNoHikage) [reply app/scripts/components/academy/academy-course.components.jsx]: (╯°□°）╯︵ ┻━┻

Comment (Aldrian) [reply app/scripts/components/academy/academy-course.components.jsx]: ┬─┬ノ( º _ ºノ)

**Pull request merged**